auth/src/commonMain/kotlin/com/liftric/auth/Configuration.kt

```diff
@@ -8,7 +8,15 @@ import io.ktor.http.HeadersBuilder
  * Configuration object for the auth handler
  * Holds all headers needed to make requests to AWS Cognito
  */
-class Configuration(origin: String, region: Region, val clientId: String) {
+class Configuration(region: Region, val clientId: String) {
+
+    /**
+     * For compatibility reasons this constructor is needed, previous Auth versions set the Origin header
+     */
```

Comment:
You could also add the `@Deprecation` annotation.

Reply:
Good point!